src/darwin/CHIPTool/CHIPTool/Info.plist

```diff
@@ -16,8 +16,28 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLName</key>
+			<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>customflow</string>
+			</array>
+		</dict>
+	</array>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>LSEnvironment</key>
+	<dict>
+		<key>CommissioningCustomFlowLedgerUrl</key>
+		<string>https://dcl.dev.dsr-corporation.com/api/modelinfo/models</string>
+		<key>CommissioningCustomFlowReturnUrl</key>
+		<string>customflow://payload</string>
```

Comment:
Android app uses `mt://modelinfo?payload=<>`. It will be great if we use the same redirect URL in both apps.  

https://github.com/project-chip/connectedhomeip/pull/12712

Reply:
Sure!